src/jdk.hotspot.agent/macosx/native/libsaproc/MacosxDebuggerLocal.m

```diff
@@ -277,8 +290,10 @@ jlong lookupByNameIncore(
 
   jlong address = 0;
 
-JNF_COCOA_ENTER(env);
-  NSString *symbolNameString = JNFJavaToNSString(env, symbolName);
+  NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
+  @try {
+
+  NSString *symbolNameString = JavaStringToNSString(env, symbolName);
```

Comment:
Is there a reason why `java.desktop` continues to use `JNFJavaToNSString`? I was looking to see how this was handled in other places, but I couldn't find an example where `JNFJavaToNSString` was converted to call a newly implemented `JavaStringToNSString`.

Reply:
As Magnus said that is in progress. Hoping it will be pushed very soon.